src/element-observation.js

```diff
@@ -60,6 +60,15 @@ export class StyleObserver {
     return this.element.style.cssText;
   }
 
+  _setProperty(style, value) {
+    let priority = null;
```

Comment:
hmm, correctly used `''` in the benchmarks :confused:.  It's been changed thanks.